scripts/esbuild/mock-doc.ts

```diff
@@ -23,8 +23,7 @@ export async function buildMockDoc(opts: BuildOptions) {
   // clear out rollup stuff and ensure directory exists
   await fs.emptyDir(outputDir);
 
-  // bundle d.ts
-  await bundleMockDocDts(opts, inputDir, outputDir);
+  const bundleDtsPromise = await bundleMockDocDts(inputDir, outputDir);
```

Comment:
I don't think we need the variable assignment here:
```suggestion
  await bundleMockDocDts(inputDir, outputDir);
```
In the rollup version, we wouldn't `await` the call to `bundleMockDocDts`, but would `await` the variable pointing to the associated promise